specs/AppLifecycle/EnvironmentVariables/AppLifecycle - Environment Variables (EV).md

```diff
@@ -281,20 +282,11 @@ Setting a **User** or **System** variable will persist that variable beyond the
 life of the process in the user profile or system profile.
 
 Unpackaged apps can use the existing native Win32 Get/SetEnvironment and
```

Comment:
Curious why the .NET APIs get links, but the Win32 ones don't?

Reply:
Oversight.

Reply:
Added.